R/rd2markdown.R

```diff
@@ -237,7 +239,7 @@ rd2markdown.dots <- function(x, fragments = c(), ...) {
 #' @exportS3Method
 #' @rdname rd2markdown
 rd2markdown.TEXT <- function(x, fragments = c(), ...) {
-  gsub("\\s+", " ", gsub("\n", " ", x))
+  trim_extra_newlines(x)
```

Comment:
The change that made the function discover standalone new line signs was a great idea, but unfortunately, there was one gap that was not possible to overcome which made me revert that. Namely, developers sometimes break the line once in roxygen documentation to make it readable and shorten lines. However if one does it just after a special documentation block (like `\code{}`) it will always appear as a solo `\n` TEXT tag and therefore would be falsely recognized as new paragraph. Therefore I decided to look for a different solution to this issue.

Reply:
Ah, good catch! I hadn't considered this. Definitely complicates things.